### PR TITLE
doc: Update supported RSA key sizes

### PR DESCRIPTION
**Pull request by vili-nordic**
* New oberon library supports RSA.

### CODE REVIEW

**Review by Vge0rge — APPROVED** [review]

**Review by mia-ko — COMMENTED** [review]
Looks good :)
Should we mention the key size update in the release notes?

File: doc/nrf/libraries/nrf_security/doc/backend_config.rst [hunk 1]
```diff
@@ -481,13 +481,13 @@ To configure Rivest-Shamir-Adleman (RSA) support, set the :kconfig:option:`CONFI
...
 ===============
 
-+-----------+-------------------+----------------------+-----------------------+
-| Algorithm | Backend           | Key size             | Note                  |
-+===========+===================+======================+=======================+
-| RSA       | cc3xx             | Limited to 2048 bits |                       |
-|           +-------------------+----------------------+-----------------------+
-|           | nrf_oberon        | N/A                  | Backend not supported |
-+-----------+-------------------+----------------------+-----------------------+
++-----------+-------------------+--------------------------------+-----------------------------------+
```
[the remark this answers is not in the document]
Comment (vili-nordic) [reply doc/nrf/libraries/nrf_security/doc/backend_config.rst:484]: We can do that

**Review by melwee01 — APPROVED** [review]

File: doc/nrf/libraries/nrf_security/doc/backend_config.rst [hunk 2]
```diff
@@ -481,13 +481,29 @@ To configure Rivest-Shamir-Adleman (RSA) support, set the :kconfig:option:`CONFI
...
+|           +-------------------+--------------+
+|           | cc312             | 1024-bit key |
+|           |                   +--------------+
+|           |                   | 1536-bit key |
+|           |                   +--------------+
+|           |                   | 2048-bit key |
+|           |                   +--------------+
+|           |                   | 3072-bit key |
+|           +-------------------+--------------+
+|           | nrf_oberon        | 1536-bit key |
```
Comment (joerchan) [inline doc/nrf/libraries/nrf_security/doc/backend_config.rst:501]: From what I can tell nrf_oberon supports 1024-bit key also.

**Review by joerchan — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/nrf_security/doc/backend_config.rst (hunk 2 above)
Comment (vili-nordic) [reply doc/nrf/libraries/nrf_security/doc/backend_config.rst]: right. added that. it also support 4k, 6k and 8k but that we have not tested that.

Comment (joerchan) [reply doc/nrf/libraries/nrf_security/doc/backend_config.rst]: True, but from 4k and up it requires a larger stack size in TF-M, so we need to do some changes to support these key sizes.

**Review by joerchan — APPROVED** [review]

**Review by mia-ko — APPROVED** [review]

**Pull request merged**